odoo/cli/module.py

```diff
@@ -0,0 +1,163 @@
+import logging
+
+import odoo
+from odoo.cli.command import Command
+from odoo.modules.loading import load_demo
+from odoo.modules.module import get_module_path, initialize_sys_path
+from odoo.modules.module_graph import ModuleGraph
+from odoo.modules.registry import Registry
+from odoo.tools import config
+
+_logger = logging.getLogger(__name__)
+
+
+class Module(Command):
+    """ Install, Upgrade, Uninstall modules, install demo data """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        subparsers = self.parser.add_subparsers(
+            dest='subcommand', required=True,
+            help='Subcommands help')
+
+        self.force_demo_parser = subparsers.add_parser(
+            'force-demo',
+            help="Install demo data (force)",
+            description="Install demonstration data (force)",
+        )
+        self.install_parser = subparsers.add_parser(
+            'install',
+            help="Install modules",
+            description="Install selected modules",
+        )
+        self.upgrade_parser = subparsers.add_parser(
+            'upgrade',
+            help="Upgrade modules",
+            description="Upgrade selected modules",
+        )
+        self.uninstall_parser = subparsers.add_parser(
+            'uninstall',
+            help="Uninstall modules",
+            description="Uninstall selected modules",
+        )
+
+        for parser in (
+            self.install_parser,
+            self.uninstall_parser,
+            self.force_demo_parser,
+            self.upgrade_parser,
+        ):
+            parser.add_argument(
+                '-c', '--config', dest='config',
+                help="use a specific configuration file")
+            parser.add_argument(
+                '-d', '--database', dest='db_name', default=None,
+                help="database name, connection details will be taken from the config file")
+
+        self.install_parser.add_argument(
+            'modules', nargs='+', metavar='MODULE',
+            help="modules to be installed")
+        self.uninstall_parser.add_argument(
+            'modules', nargs='+', metavar='MODULE',
+            help="modules to be uninstalled")
+        self.upgrade_parser.add_argument(
+            'modules', nargs='+', metavar='MODULE',
+            help="modules to be upgraded")
+        self.upgrade_parser.add_argument(
+            'modules', nargs='+', metavar='MODULE',
+            help="modules to be upgraded")
+        self.force_demo_parser.add_argument(
```

Comment:
If we support `--force-demo account`, we also need to support `--force-demo all` which might be the most common command for developers forget to use `--with-demo`